### DISCUSSION TITLE
Filter by codeowners manually

### DISCUSSION THREAD

**Discussion by nickofthyme** [community/community#118339] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I like the option in pull requests that allows me to view only the files that I own per the [`CODEOWNERS`](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file, as seen below...

<img width="317" alt="image" src="https://github.com/community/community/assets/19007109/770691a9-7cf9-42f2-a605-2663523c87a6">

But there are times when I'd like to view all the files for a team that I am not apart of so I can ping them to say "the changes are minor", or "only a single line change", to incentivize a faster response. But currently I cannot do this quickly, the best way is to just go through each file change to see what has changes per codeowner.

<img width="448" alt="image" src="https://github.com/community/community/assets/19007109/d38646aa-5cf0-48a3-9989-95ea2ab58b29">

Ideally, there is a way to filter the files for all applicable codeowners teams.

**Comment by YossiSaadi** [comment]
Hey @nickofthyme :)
I had the same issue as you had
So I've created a Chrome extension for it
check out https://github.com/YossiSaadi/github-codeowners-filter-chrome-extension
waiting for Chrome Web Store to approve it 🤞🏼

**Reply by YossiSaadi** [reply]
https://chromewebstore.google.com/detail/github-code-owners-filter/jhbeenhmiadkocfjpmipapdnpgngbmmo

**Reply by jhasbani-clickup** [reply]
Great idea! I really want this! It doesn't seem to be working for me. I am not seeing your additions to the filter. Perhaps a change was made in github since you last published.
